recipes/iqa/all/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MSVC AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
I believe Conan does this automatically

Reply:
Conan doesn't inject this setting.

Reply:
Even better I just found this gem

https://github.com/conan-io/conan-center-index/pull/2367#discussion_r464125707

Reply:
Indeed Conan doesn't do it. The best approach is asking for the upstream providing it.

Reply:
IQA is unmaintained now.